# HG changeset patch
# Parent  705870b5f0541e60e48de30f933cad835850c0f4
Log something before forking into the background

This will hopefully avoid some confusion when forgetting -f.

diff --git a/core/lightsd.c b/core/lightsd.c
--- a/core/lightsd.c
+++ b/core/lightsd.c
@@ -232,8 +232,11 @@
         lgtd_err(1, "can't setup lightsd");
     }
 
-    if (!lgtd_opts.foreground && !lgtd_daemon_unleash()) {
-        lgtd_err(1, "can't fork to the background");
+    if (!lgtd_opts.foreground) {
+        lgtd_info("forking into the background now...");
+        if (!lgtd_daemon_unleash()) {
+            lgtd_err(1, "can't fork to the background");
+        }
     }
 
     lgtd_lifx_watchdog_start_discovery();
